test/bench/100_pods/S3/cleaner.py

```diff
@@ -0,0 +1,30 @@
+import json
```

Comment:
You need a shebang here:
```suggestion
#! /usr/bin/env python3

import json
```

With it, if your file is executable, you would be able to launch it with `./cleaner.py`

Reply:
done ! thank you :)